frontend/src/components/MainToolbar.vue

```diff
@@ -199,12 +220,29 @@ export default {
         query
       }
     },
-    darkMode: {
+    colorScheme: {
       get () {
-        return this.$store.state.darkMode
+        switch (this.$store.state.colorScheme) {
+          case 'light':
+            return 0
+          case 'dark':
+            return 1
+          default:
+            return 2
+        }
       },
       set (value) {
-        this.setDarkMode(value)
+        switch (value) {
+          case 0:
+            this.setColorScheme('light')
+            break
+          case 1:
+            this.setColorScheme('dark')
+            break
+          default:
+            this.setColorScheme('auto')
+            break
+        }
```

Comment:
use constants instead of magic/index numbers

Reply:
done